### PR TITLE
Theme fixes

### PR DESCRIPTION
**Pull request by arschmitz**
Fixes jquery/download.jqueryui.com#335
Ref https://forum.jquery.com/topic/checkboxradio-widget-checkbox-click-doesn-t-work-with-ui-lightness-theme-22-9-2016

### CODE REVIEW

**Comment by jzaefferer** [comment]
Diff looks good to me.

**Comment by scottgonzalez** [comment]
This now also fixes https://bugs.jqueryui.com/ticket/15059.

File: themes/base/theme.css [hunk 1]
```diff
@@ -119,7 +119,7 @@ a.ui-button:active,
 .ui-icon-background,
 .ui-state-active .ui-icon-background {
 	border: #003eff/*{borderColorActive}*/;
-	background-color: #ffffff/*{fcActive}*/;
+	background-color: #ffffff/*{bgColorContent}*/;
```
Comment (scottgonzalez) [inline themes/base/theme.css]: Why are we pairing the content background with the active border? Why is this not using the active background?

Comment (scottgonzalez) [inline themes/base/theme.css]: Ok. I think it makes sense for these to match. Though it's a shame the active colors don't work here.

**Pull request closed without merging**